presto-hive-hadoop2/src/test/java/io/prestosql/plugin/hive/TestHive.java

```diff
@@ -65,4 +66,18 @@ public void testGetPartitionSplitsTableOfflinePartition()
 
         super.testGetPartitionSplitsTableOfflinePartition();
     }
+
+    @Override
+    public void testHideDeltaLakeTables()
+    {
+        assertThatThrownBy(super::testHideDeltaLakeTables)
```

Comment:
assertion on assertion....
Maybe nicer approach would be to have `bool supportsDeltaLakeTablesHiding()` in `AbstractTestHive` and override it in subclasses accordingly.